.github/workflows/all_green_check.yml

```diff
@@ -0,0 +1,46 @@
+---
+name: all_green
+
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - labeled
+      - unlabeled
```

Comment:
I dont really like having `changelog` workflow in this file, because this is the only workflow impacted by the `labeled` and `unlabeled` triggers. You should set it into a dedicated file and update triggers from this file, otherwise each a PR is labeled/unlabeled all jobs will be rerun knowing that this is not necessary for unit, sanity, and linters jobs